Doc/library/sqlite3.rst

```diff
@@ -1140,6 +1140,15 @@ Blob Objects
       end).
 
 
+.. _sqlite3-prepareprotocol-objects:
+
```

Comment:
```suggestion
.. _sqlite3-prepareprotocol-objects:

PrepareProtocol Objects
-----------------------
```

I'm guessing this doesn't belong under the `Blob Objects` section, and it doesn't really make sense to have a `ref` ref target, since a `class` ref target is already available for the class.

Reply:
Oh, that's a good point. Each of the sqlite3 classes have such (useless) ref; we can remove all of them.

Reply:
Could probably include the exceptions anchor in that clean-up as well.